CONTRIBUTORS.md

```diff
@@ -35,3 +35,4 @@
 * [SolomonSklash](https://github.com/SolomonSklash)
 * [l4yton](https://github.com/l4yton)
 * [xfgusta](https://github.com/xfgusta)
+* [adilsoybali](https://github.com/adilsoybali)
```

Comment:
This list should be kept in alphabetically sorted order, can you make that change?